01-lesson_notebooks/02b-Variables_and_Assignment.ipynb

```diff
@@ -55,90 +55,134 @@
    ]
   },
   {
+   "attachments": {},
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "Variable names should be descriptive of their purpose to enhance the readability of your code. They must start with a letter or an underscore - subsequent characters can be numbers, letters or underscores. Variable names are case sensitive - radius, Radius, raDius, etc. would be interpreted as different variables. A variable name cannot be the same as a keyword. Python 3 has the following keywords:"
+    "Variable names should be descriptive of their purpose to enhance the readability of your code. They must start with a letter or an underscore - subsequent characters can be numbers, letters or underscores. Variable names are case sensitive - radius, Radius, raDius, etc. would be interpreted as different variables. A variable name cannot be the same as a keyword. Python3 has the following keywords:"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "    False class finally is return\n",
-    "    True continue for lambda try\n",
-    "    None def from nonlocal while\n",
-    "    and del global not with\n",
-    "    as elif if or yield\n",
-    "    assert else import pass\n",
-    "    break except in raise"
+    "- | - | - | - | -\n",
+    "---|---|---|---|---\n",
+    "False | class | finally | is | return\n",
+    "True | continue | for | lambda|  try\n",
+    "None | def | from | nonlocal | while\n",
+    "and | del | global| not | with\n",
+    "as | elif | if | or | yield\n",
+    "assert | else | import | pass\n",
+    "break | except | in | raise\n"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
     "## Static Typing vs. Dynamic Typing\n",
+    "### Dynamic Typing\n",
     "\n",
     "Python is a dynamically typed language, meaning variables can change from type to type with any new assignment. For example, this is a legal statement in python:\n",
+    "```python\n",
+    "my_number = 57\n",
     "\n",
-    "`my_number = 57`\n",
-    "\n",
-    "`my_number = 'fifty-seven'`\n",
+    "my_number = 'fifty-seven'\n",
```

Comment:
maybe add a short sentence explaining 'fifty-seven' and '57' are different and my_number cannot be both at once.  Up to you  really but it 's just to clear confusion for beginner. If you think it 's not needed, then it 's not needed